packages/office-ui-fabric-react/src/utilities/color/cssColor.ts

```diff
@@ -0,0 +1,104 @@
+import { IRGB } from './interfaces';
+import { MAX_COLOR_ALPHA } from './consts';
+import { COLOR_VALUES } from './colorValues';
+import { hsl2rgb } from './hsl2rgb';
+
+/**
+ * Converts a valid CSS color string to an RGB color.
+ * Note that hex colors *must* be prefixed with # to be considered valid.
+ * Alpha in returned color defaults to 100.
+ */
+export function cssColor(color: string): IRGB | undefined {
+  if (!color) {
+    return undefined;
+  }
+  return _named(color) || _hex3(color) || _hex6(color) || _rgba(color) || _hsla(color);
+}
+
+/**
+ * If `str` is a valid HTML color name, returns an RGB color (with alpha 100).
+ * Otherwise returns undefined.
+ */
+function _named(str: string): IRGB | undefined {
+  const c = (COLOR_VALUES as any)[str.toLowerCase()];
```

Comment:
I'm pretty sure there is a way to avoid any here, @KevinTCoughlin do you know a magic typescript way of keeping the type here?

Reply:
All this code is copied and pasted, all I did is move it. Improvements can/should be made in a subsequent PR.

Reply:
`any` is hard to avoid in this case, since the string could be literally anything (a valid key of COLOR_VALUES, a hex string, etc)

Reply:
You can avoid the any if you do `COLOR_VALUES[str.toLowerCase() as COLOR_VALUES];`

Reply:
Wouldn't that be `as keyof COLOR_VALUES`? Either way it's not a huge deal IMO.

Reply:
No because color values is not an object but an enum. An enum does not have the kind of keys objects do in the type layer :)